src/flash.rs

```diff
@@ -93,13 +93,14 @@ impl FlashError {
     /// Helper method to clear error bits raised
     pub fn clear(&self, sr: &mut SR) {
         match self {
-            FlashError::WriteProtect => sr.inner().modify(|_,w| w.wrperr().set_bit()),
-            FlashError::Alignment => sr.inner().modify(|_,w| w.pgaerr().set_bit()),
-            FlashError::Size => sr.inner().modify(|_,w| w.sizerr().set_bit()),
-            FlashError::ReadProtection => sr.inner().modify(|_,w| w.rderr().set_bit()),
-            FlashError::NotZero => sr.inner().modify(|_,w| w.notzeroerr().set_bit()),
-            FlashError::FetchAbort => sr.inner().modify(|_,w| w.fwwerr().set_bit()),
-            _ => {},
+            FlashError::WriteProtect => sr.inner().modify(|_, w| w.wrperr().set_bit()),
+            FlashError::Alignment => sr.inner().modify(|_, w| w.pgaerr().set_bit()),
+            FlashError::Size => sr.inner().modify(|_, w| w.sizerr().set_bit()),
+            FlashError::ReadProtection => sr.inner().modify(|_, w| w.rderr().set_bit()),
+            FlashError::NotZero => sr.inner().modify(|_, w| w.notzeroerr().set_bit()),
+            FlashError::FetchAbort => sr.inner().modify(|_, w| w.fwwerr().set_bit()),
+            FlashError::Locked => ({}),
```

Comment:
`FlashError::FetchAbort | FlashError::Locked => {},`

Reply:
Did you mean `FlashError::Locked | FlashError::Unknown`? If so, done https://github.com/thenewwazoo/stm32l0x1-hal/pull/1/commits/626a746037b4136b593e15f1af230406aa46413b.